maps/random_ruins/exoplanets/moghes/moghes_bar.dm

```diff
@@ -0,0 +1,152 @@
+/datum/map_template/ruin/exoplanet/moghes_bar
+	name = "Moghes Inn"
+	id = "moghes_bar"
+	description = "An inn for the weary traveler, located in the Untouched Lands of Moghes"
+	spawn_weight = 1
+	spawn_cost = 2
+	template_flags = TEMPLATE_FLAG_NO_RUINS|TEMPLATE_FLAG_RUIN_STARTS_DISALLOWED
+	sectors = list(SECTOR_UUEOAESA)
+	prefix = "moghes/"
+	suffixes = list("moghes_bar.dmm")
+	ban_ruins = list("ouerea_bar")
+
+/area/moghes_bar
+	name = "Moghes Bar"
+	icon_state = "bar"
+	requires_power = FALSE
+	dynamic_lighting = TRUE
+	no_light_control = FALSE
+	base_turf = /turf/simulated/floor/exoplanet/grass/moghes/dirt
+	area_flags = AREA_FLAG_RAD_SHIELDED|AREA_FLAG_INDESTRUCTIBLE_TURFS
+	area_blurb = "The sound of music and the warmth of a fire fill this place."
+
+/datum/ghostspawner/human/moghes_bar_innkeeper
+	name = "Moghresian Innkeeper"
+	short_name = "moghes_bar_innkeeper"
+	desc = "Operate an inn on Moghes"
+	tags = list("External")
+	welcome_message = "You are the proprietor of an inn on Moghes. Serve drinks and food, rent your rooms, and make a profit."
+
+	extra_languages = list(LANGUAGE_UNATHI, LANGUAGE_AZAZIBA)
+	max_count = 1
+	spawnpoints = list("moghes_bar_innkeeper")
+	possible_species = list(SPECIES_UNATHI)
+	allow_appearance_change = APPEARANCE_PLASTICSURGERY
+	outfit = /obj/outfit/admin/moghes_bar_innkeeper
+
+	assigned_role = "Moghresian Innkeeper"
+	special_role = "Moghresian Innkeeper"
+	respawn_flag = null
+	uses_species_whitelist = FALSE
+
+/obj/outfit/admin/moghes_bar_innkeeper
+	name = "Moghes Innkeeper"
+	uniform = /obj/item/clothing/under/unathi
+	suit = /obj/item/clothing/suit/unathi/robe/robe_coat/blue
+	shoes = /obj/item/clothing/shoes/sandals
+	back = /obj/item/storage/backpack/satchel/leather
+	r_pocket = /obj/item/storage/wallet/random
+	l_ear = null
+	id = /obj/item/card/id
+
+/obj/outfit/admin/moghes_bar_inkeeper/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	var/uniform_color = "[pick("#473b13", "#134718", "#0a6959")]"
+	if(H?.w_uniform)
+		H.w_uniform.color = uniform_color
+	if(H?.shoes)
+		H.shoes.color = "#423509"
```

Comment:
Might aswell return early if no human is passed in as parameter, also useless var